location-service/src/test/java/uk/gov/cshr/locationservice/service/LocationServiceControllerTest.java

```diff
@@ -62,7 +62,7 @@ public void testPartialPostcode() throws Exception {
     @Test
     public void testFullPostcode() throws Exception {
 
-        Coordinates coordinates = findCoordinates("BS16JS");
+        Coordinates coordinates = findCoordinates("BS1 6JS");
```

Comment:
Hey
if you put BS16JS in azure deployed loc-service
does it not come with results
we had one job posted for bristol yesterday in Azure
can you test with this postcode if the results come
basically what i am trying to see that padding the postcode with space in test class is good or the modifying the actual code with padded space before calling google service